src/astro/constants.py

```diff
@@ -3,3 +3,8 @@
 DEFAULT_CHUNK_SIZE = 1000000
 PYPI_PROJECT_NAME = "astro-projects"
 DEFAULT_SCHEMA = "tmp_astro"
+POSTGRES = "postgres"
```

Comment:
This is looking great, what do you think about using an `Enum` to group them?

Reply:
I tried to update it, and got carried away with this refactor updated all the instances of string dbs was raising lot of issues on my end, I'll address this as part of the a refactor ticket.